sympy/geometry/tests/test_parabola.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import division
+
+from sympy import Dummy, Rational, S, Symbol, pi, sqrt, oo
+from sympy.core.compatibility import range
+from sympy.geometry import (Circle, Ellipse, GeometryError, Line, Point,
+                            Point2D, Polygon, Ray, RegularPolygon,
+                            Segment, Triangle, intersection, Parabola)
+from sympy.integrals.integrals import Integral
+from sympy.utilities.pytest import raises, slow
```

Comment:
It seems that many of the above imports are not necessary for these tests. They should be removed.